examples/jsm/renderers/CSS2DRenderer.js

```diff
@@ -206,8 +206,11 @@ class CSS2DRenderer {
 
 				}
 
-				const distanceA = cache.objects.get( a ).distanceToCameraSquared;
-				const distanceB = cache.objects.get( b ).distanceToCameraSquared;
+				const cacheDataA = cache.objects.get( a );
```

Comment:
Um, instead of adding a change here, how about updating `filterAndFlatten()` and only add visible objects in the result array?

Reply:
In `filterAndFlatten ` should consider it's an ancestors ,i don't have a good idea at the moment.
 Can you give me some advice?

Reply:
I think it should work if you use `traverseVisible()` in `filterAndFlatten()`.

Reply:
Good idea!